### PR TITLE
docs(ui/theme): add tip on using dark parent themes on Android

### LINKED ISSUE #1281: When using the dark color scheme, make sure that you are using dark parent themes on Android
### Please, tell us what's the problem?
[ ] A typo
[ ] Wrong documentation
[x] Improvement of existing article
[ ] Missing documentation
[ ] New article needed

### Please, tell us more details

https://docs.nativescript.org/angular/ui/theme#color-schemes should clarify something like this:

“When using the dark color scheme, make sure that you are using dark parent themes on Android:

In `app/App_Resources/Android/src/main/res/values/styles.xml` and in `app/App_Resources/Android/src/main/res/values-v21/styles.xml` remove `“Light.”` from parent namespaces, e.g. `Theme.AppCompat.Light.NoActionBar` becomes `Theme.AppCompat.NoActionBar`  etc.”

### PR DESCRIPTION
**Pull request by yavulan**
## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [x] If there is an issue related with this PR, point it out here #1281.

## What is the current state of the documentation article?

Good.
## What is the new state of the documentation article?

See #1281.

Closes #1281.

### CODE REVIEW

File: docs/ui/theme.md [hunk 1]
```diff
@@ -65,7 +65,10 @@ And in order to use the dark scheme, add this line of code to your `app.css` ins
 @import '~nativescript-theme-core/css/core.dark.css';
 ```
 
-> **TIP**: When using the dark color scheme, make sure that you, 1) apply the [`page` class name](#page) to ensure your app’s text displays appropriately, and 2) [alter your app’s status bar colors](#status-bar-considerations).
+> **TIP**: When using the dark color scheme, make sure that you:
+> * apply the [`page` class name](#page) to ensure your app’s text displays appropriately;
+> * [alter your app’s status bar colors](#status-bar-considerations);
+> * use dark parent themes on Android. In `style.xml` files in the `App_Resources\Android\values[-v21]` folders remove `Light.` from “parent” attributes (e.g., `Theme.AppCompat.Light.NoActionBar` becomes `Theme.AppCompat.NoActionBar`).
```
Comment (tsonevn) [inline docs/ui/theme.md:71]: I think that the last step is not needed. I tested the description, and the first two steps seem to be enough for changing the theme while using `nativescript-theme-core`plugin.

**Review by tsonevn — CHANGES_REQUESTED** [review]

**Comment by yavulan** [comment]
At least `Dialog` and `Datepicker` not being rendered as expected without the last step. And, as I understand, there is no way to set some of required for dark theme styles from CSS for these ui elements for now. See https://github.com/NativeScript/NativeScript/issues/1903, https://github.com/NativeScript/NativeScript/issues/3830.

File: docs/ui/theme.md (hunk 1 above)
Comment (yavulan) [reply docs/ui/theme.md:71]: At least `Dialog` and `Datepicker` not being rendered as expected without the last step. And, as I understand, there is no way to set some of required for dark theme styles from CSS for these ui elements for now. See https://github.com/NativeScript/NativeScript/issues/1903, https://github.com/NativeScript/NativeScript/issues/3830.

Comment (tsonevn) [reply docs/ui/theme.md:71]: 👍

**Review by tsonevn — APPROVED** [review]

**Pull request merged**